src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/reissuefurtherevidence/ReissueFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -59,6 +59,14 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                     .flatMap(x -> x == null ? null : x.stream())
                     .filter(f -> selectedDocumentUrl.get().equals(f.getValue().getDocumentLink().getDocumentUrl()))
                     .findFirst();
+            if (optionalSelectedDocument.isEmpty()) {
+                optionalSelectedDocument  = Stream
+                        .of(sscsCaseData.getSscsDocument(), sscsCaseData.getSscsWelshDocuments())
+                        .flatMap(x -> x == null ? null : x.stream())
```

Comment:
can we use more meaningful variable names like document for clarity

Reply:
changed